types/system-task/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for system-task 1.0
+// Project: https://github.com/leocwlam/system-task
+// Definitions by: Leo Lam <https://github.com/leocwlam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
+
+/// <reference types="node" />
+
+export function asyncProcess(items: any[], executeAsyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
+export function syncProcess(items: any[], executeSyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
+
+export default class SystemTask {
```

Comment:
```suggestion
export = SystemTask;
class SystemTask {
```

Reply:
I have the new PR with your suggestion.
Thanks again.